mappings/net/minecraft/client/audio/Sound.mapping

```diff
@@ -1 +1,4 @@
 CLASS none/cab net/minecraft/client/audio/Sound
+	METHOD e getWeight ()I
+	METHOD f getCopy ()Lnone/cab;
```

Comment:
Does this literally `get` some preexisting copy/clone or does it create one when the method is called? If the latter, it shouldn't be prefixed with `get` imo.